pkg/hubble/metrics/metrics.go

```diff
@@ -40,7 +48,21 @@ func ProcessFlow(ctx context.Context, flow *pb.Flow) error {
 	return nil
 }
 
-// initMetrics initialies the metrics system
+func ProcessPodDeletion(pod *slim_corev1.Pod) error {
```

Comment:
Similar to @christarazi 's comment, I wonder if this could be a method on some object instead, and then the queue would be a field on a struct rather than a global.